Ginger/Ginger/Actions/ActionEditPages/ValidationDBPage.xaml.cs

```diff
@@ -385,11 +421,12 @@ private void SetVisibleControlsForAction()
                             TableColWhereStackPanel.Visibility = Visibility.Visible;
                             txtWhere.Visibility = Visibility.Collapsed;
                             lblWhere.Visibility = Visibility.Hidden;
-                            TableColWhereStackPanel.Height = 70;
+                            TableColWhereStackPanel.Height = 40;
                             DoCommit.Visibility = Visibility.Collapsed;
                             FreeSQLStackPanel.Visibility = Visibility.Collapsed;
                             DoUpdate.Visibility = Visibility.Visible;
                             RadioButtonsSection.Visibility = Visibility.Collapsed;
+                            CosmosParametersGrid.Visibility = Visibility.Visible;
```

Comment:
rename the Grid as UpdateDBParamGrid
Currently it is used only for Cosmos but in future it might be reused for other DB Types